src/file_organizer/api/routers/config.py

```
@@ -1,163 +1,153 @@
+        api_error_response(
+            409,
+            error="unsupported_config_version",
+            message="On-disk profile schema version is unsupported for safe overwrite",
+        ),
         validation_error_response(),
     ),
 )
 def update_config(
     request: ConfigUpdateRequest,
-    settings: ApiSettings = Depends(get_settings),
+    manager: ConfigManager = Depends(get_config_manager),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)